scipy/linalg/flapack.pyf.src

```diff
@@ -2788,6 +2788,100 @@ subroutine <prefix2>sbevx(ab,ldab,compute_v,range,lower,n,kd,q,ldq,vl,vu,il,iu,a
 end subroutine <prefix2>sbevx
 
 
+subroutine <prefix2>stebz(d,e,range,vl,vu,il,iu,tol,order,n,work,iwork,m,nsplit,w,iblock,isplit,info)
+  ! computes all or selected eigenvalues of a real, symmetric tridiagonal
+  ! matrix.
+
+  callstatement (*f2py_func)((range>0?(range==1?"V":"I"):"A"),order,&n,&vl,&vu,&il,&iu,&tol,d,e,&m,&nsplit,w,iblock,isplit,work,iwork,&info)
+  callprotoargument char*,char*,int*,<ctype2>*,<ctype2>*,int*,int*,<ctype2>*,<ctype2>*,<ctype2>*,int*,int*,<ctype2>*,int*,int*,int*,int*,int*
+
+  <ftype2> dimension(n),intent(in) :: d
+  <ftype2> dimension(n-1),intent(in) :: e
+  <ftype2> intent(in) :: vl
+  <ftype2> intent(in) :: vu
+  character intent(in) :: order
+  integer intent(in) :: il
+  integer intent(in) :: iu
+  <ftype2> intent(in) :: tol
+  integer intent(in) :: range
+  integer depend(d),intent(hide) :: n = shape(d,0)
+  <ftype2> dimension(MAX(1,4*n)),depend(n),intent(hide) :: work  ! The dimension of work must be at least max(1, 4n).
```

Comment:
Nitpick: [LAPACK's documentation](http://www.netlib.org/lapack/explore-html/d2/d24/group__aux_o_t_h_e_rcomputational_ga28f88843da09a0ee400daf46caaabec6.html#ga28f88843da09a0ee400daf46caaabec6) says
> dimension (4*N)

(without _at least_). The current wording suggests that the routine may profit from a larger `work`.